src/search-param-url-generator.ts

```diff
@@ -101,6 +101,14 @@ export class SearchParamURLGenerator {
       params.append('debugging', 'true');
     }
 
+    if (searchParams.uid) {
+      params.append('uid', searchParams.uid);
+    }
+
+    if (searchParams.includeClientUrl !== false) {
+      params.append('client_url', window.location.href);
```

Comment:
Q: what is this used for?

Reply:
This is so the PPS has a view of what client URL params resulted in a given request, to aid with debugging issues on the PPS side. I'll add a more descriptive doc comment for this.

Reply:
Q: will it only. be in play when debugging is in play?

Reply:
It should be in play at all times (regardless of the debugging param).